okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -271,8 +271,7 @@ SSLSocketFactory createSocketFactory() {
       case TLS:
         try {
           if (sslSocketFactory == null) {
-            SSLContext sslContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
-            sslContext.init(null, null, null);
+            SSLContext sslContext = SSLContext.getInstance("Default", Platform.get().getProvider());
```

Comment:
It's somewhat circuitously documented here

https://docs.oracle.com/javase/7/docs/api/javax/net/ssl/SSLContext.html#getDefault()